### ISSUE TITLE
Error: Dynamic require of "crypto" is not supported

### ISSUE THREAD

**Issue by basbroens** [vuejs/devtools-next#288]
When my page is loading the stand alone app works, ,but when the page is loaded completely i get:

Waiting for connection.... 

And i the browser i get: 

Error: Dynamic require of "crypto" is not supported:

```
  var yI = Object.create;
    var Vh = Object.defineProperty;
    var bI = Object.getOwnPropertyDescriptor;
    var EI = Object.getOwnPropertyNames;
    var TI = Object.getPrototypeOf
      , _I = Object.prototype.hasOwnProperty;
    var SI = (t=>typeof require != "undefined" ? require : typeof Proxy != "undefined" ? new Proxy(t,{
        get: (e,s)=>(typeof require != "undefined" ? require : e)[s]
    }) : t)(function(t) {
        if (typeof require != "undefined")
            return require.apply(this, arguments);
        throw Error('Dynamic require of "' + t + '" is not supported')
    });

```
How do i fix this?

**Comment by duowb** [comment]
Can you provide a minimum reproduction? 😉

**Comment by alexzhang1030** [comment]
Can you share your environment?

**Comment by dmitryuk** [comment]
The same issue

**Comment by dmitryuk** [comment]
> Can you provide a minimum reproduction? 😉
```
<head>
<script src="http://localhost:8098"></script>
</head>
<body>
<div id=app></div>
</body>
```

**Comment by webfansplz** [comment]
Fixed in V7.0.24
